.buildkite/pipeline.yml

```
@@ -336,104 +336,208 @@ steps:
   - group: "TurbulenceConvection"
     steps:
 
-      - label: ":balloon: Single column EDMF Bomex"
-        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --config column --FLOAT_TYPE Float64 --hyperdiff false --moist equil --subsidence Bomex --edmf_coriolis Bomex --ls_adv Bomex --turbconv edmf --turbconv_case Bomex --split_ode false --ode_algo ODE.Euler --dt_save_to_sol 5mins --z_elem 60 --z_stretch false --z_max 3e3 --job_id edmf_bomex --dt 6secs --t_end 6hours --regression_test true --anelastic_dycore true --apply_limiter false --debugging_tc true --dt_save_to_disk 5mins"
-        artifact_paths: "edmf_bomex/*"
+      - label: ":balloon: SCM EDMF GABLS (Dry)"
+        command: "julia --color=yes --project=examples examples/hybrid/driver.jl \
+--config column --turbconv edmf --debugging_tc true --regression_test true \
+--FLOAT_TYPE Float64 --hyperdiff false --apply_limiter false \
+--turbconv_case GABLS --edmf_coriolis GABLS \
+--z_elem 8 --z_stretch false --z_max 400 \
+--t_end 9hours --dt_save_to_sol 5mins --dt_save_to_disk 5mins \
+--moist dry \
+--job_id compressible_edmf_gabls --dt 4secs"
```

I *think* the following should work:

```suggestion
        command: >
          julia --color=yes --project=examples examples/hybrid/driver.jl
          --config column --turbconv edmf --debugging_tc true --regression_test true
          --FLOAT_TYPE Float64 --hyperdiff false --apply_limiter false
          --turbconv_case GABLS --edmf_coriolis GABLS
          --z_elem 8 --z_stretch false --z_max 400
          --t_end 9hours --dt_save_to_sol 5mins --dt_save_to_disk 5mins
          --moist dry
          --job_id compressible_edmf_gabls --dt 4secs
```